core/unit_test/TestNonTrivialScalarTypes.hpp

```diff
@@ -140,6 +140,11 @@ struct array_reduce {
 
   KOKKOS_INLINE_FUNCTION
   array_reduce &operator=(const array_reduce &src) {
+    // ROCm 5.5 and earlier returns the wrong result when early return is enable
+#if !defined(KOKKOKS_ENABLE_HIP) || (HIP_VERSION_MAJOR > 5) || \
```

Comment:
There is a typo here: `KOKKOKS_ENABLE_HIP` -> `KOKKOS_ENABLE_HIP`...

Reply:
Great...
I looked at the guards this morning and didn't see it.  Now that you highlighted it it is obvious :/

Reply:
Not sure if that can be handled by a spellcheck 
https://github.com/marketplace/actions/github-spellcheck-action